docs/05-local-execution.ipynb

```diff
@@ -130,7 +130,7 @@
     "        \"what is the time in london?\",\n",
     "        \"I live in Rome, what time is it?\",\n",
     "    ],\n",
-    "    function_schema=time_schema,\n",
+    "    function_schemas=[time_schema],\n",
```

Comment:
I think it would preferred to keep the parameter name the same `function_schema` and allow it to consume a single schema *or* a list of schemas

Reply:
Don't we typically use plural for lists? Even if they're of length 1?

Then when looping through the lists (if we ever need to) we can do 

```
for function_schema in function_schemas
```

and it's clear which is the iteration and which is the array.